tests/functional/tests/mon/test_mons.py

```diff
@@ -0,0 +1,29 @@
+
+class TestMons(object):
+
+    def test_ceph_mon_package_is_installed(self, CephNode, Package):
+        assert Package("ceph-mon").is_installed
+
+    def test_mon_listens_on_6789(self, CephNode, Socket):
+        assert Socket("tcp://%s:6789" % CephNode["address"]).is_listening
+
+    def test_mon_service_is_running(self, CephNode, Service):
+        service_name = "ceph-mon@ceph-%s" % CephNode["vars"]["inventory_hostname"]
+        assert Service(service_name).is_running
+
+    def test_mon_service_is_enabled(self, CephNode, Service):
+        service_name = "ceph-mon@ceph-%s" % CephNode["vars"]["inventory_hostname"]
+        assert Service(service_name).is_enabled
+
+    def test_can_get_cluster_health(self, CephNode, Command):
+        output = Command.check_output("sudo ceph -s")
```

Comment:
This only shows a response from a monitor, we actually don't know Ceph health state from this. Moreover this won't work with clusters with names different than 'ceph'.
I think we should use `ceph --connect-timeout 5 --cluster $CLUSTER_NAME health` and look for `HEALTH_OK`. And we also fail after 5 seconds if the mon didn't respond.